persistent/Database/Persist/Sql/Orphan/PersistQuery.hs

```diff
@@ -366,6 +373,8 @@ filterClauseHelper includeTable includeWhere conn orNull filters =
         name =
             (if includeTable
                 then ((tn <> ".") <>)
+                else if includeExcluded -- need this for PostgreSQL queries
```

Comment:
here's where I pass in the `includeExcluded` param; I tried to follow the same pattern as `includeTable`; does this seem to make sense?  All the tests that use this code still pass since the param is only used in `filterClauseWithVals`.